QNormalizModule.cpp

```diff
@@ -453,7 +446,7 @@ struct NumberFieldCone{
 void delete_cone_renf( PyObject* cone ){
   NumberFieldCone * cone_ptr = reinterpret_cast<NumberFieldCone*>( PyCapsule_GetPointer( cone, cone_name ) );
   delete cone_ptr->cone;
-  delete cone_ptr->nf;
+  //delete cone_ptr->nf;
```

Comment:
Are you sure `nf` should not be deleted with the cone? It gets allocated in the cone constructor, see https://github.com/sebasguts/PyQNormaliz/blob/master/QNormalizModule.cpp#L542
Not deleting this would cause a memory leak.

Reply:
I am not sure about this, but I *am* getting "pointer being freed was not allocated" without this patch. But since other things go wrong with the memory management that I have not been able to pin down, this may be a fix for the wrong thing.

Reply:
Could you post the valgrind output, too? I am not able to compile the proper sage branch, and as you said, this does not happen in pure python.